File: C++/14_friends_and_money_Intermediate.cpp
#include <iostream>
using namespace std;

int main() {
    int T;
    cin >> T;
    while (T--) {
        int X, Y, Z;
        cin >> X >> Y >> Z;
        if (X == Y + Z || Y == Z + X || Z == X + Y)
            cout << "yes" << endl;
        else
            cout << "no" << endl;
    }
    return 0;
}

/**********************************************************
 * @INFO
 * Code by Ashish Singh
 * @INFO
 * Gmail - ashishlodhi5559@gmail.com
 * @INFO
 * Github - AshishBytes
 * @INFO
 * LinkedIn - ashishbytes
 * @INFO
 * Instagram - @itz_ash._u
 * @INFO
 * Twitter - @itz_ash_u
 * @INFO
 **********************************************************/

File: Python/14_friends_and_money_Intermediate.py
T = int(input())
for _ in range(T):
    X, Y, Z = map(int, input().split())
    if X == Y + Z or Y == Z + X or Z == X + Y:
        print("yes")
    else:
        print("no")

##########################################################
# @INFO
# Code by Ashish Singh
# @INFO
# Gmail - ashishlodhi5559@gmail.com
# @INFO
# Github - AshishBytes
# @INFO
# LinkedIn - ashishbytes
# @INFO
# Instagram - @itz_ash._u
# @INFO
# Twitter - @itz_ash_u
# @INFO
##########################################################
